cvx/markowitz/risk/sample/sample.py

```diff
@@ -47,7 +47,7 @@ def estimate(self, variables: Dict[str, cp.Variable]) -> cp.Expression:
             )
         )
 
-    def update(self, **kwargs):
+    def update(self, **kwargs: Matrix) -> None:
```

Comment:
Same as above.

Reply:
https://adamj.eu/tech/2021/05/11/python-type-hints-args-and-kwargs/

Reply:
Interesting, thanks for sharing!